src/main/java/com/epam/university/java/core/task001/Task001Impl.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.university.java.core.task001;
+
+public class Task001Impl implements Task001 {
```

Comment:
There is no need to create PR for me, just merge your code to your own repository to the `master` branch.